pkg/ruler/ruler.go

```diff
@@ -530,10 +530,17 @@ func (r *Ruler) getLocalRules(userID string) ([]*GroupStateDesc, error) {
 
 	for _, group := range groups {
 		interval := group.Interval()
+
+		// The mapped filename is base64 encoded to make handling `/` characters easier
```

Comment:
Please update this comment (it's not base64 anymore)

Reply:
👍